controllers/recipes.js

```diff
@@ -184,14 +190,31 @@ module.exports = {
           },
         },
       ];
-      const searchResults = await Recipe.aggregate(searchParams);
+
+      const searchResults = await Recipe.aggregate(searchParams)
+        .sort({ createdAt: "desc" })
+        .skip(skip)
+        .limit(limit);
+
+      const totalRecipes = searchResults.length;
+      console.log(`Total # of queried recipes = ${totalRecipes}`)
+
       console.log(
         "✅ You have successfully performed a search (i.e. calling Recipe.aggregate did not blow up)."
       );
       console.log("📜 Your search parameters were:");
       console.log(JSON.stringify(searchParams, null, 2)); // From https://stackoverflow.com/a/10729391
       console.log("🎁 ...and your search results are:");
       console.log(searchResults);
+      res.render("searchResults.ejs", {
+        user: req.user,
+        userEnteredSearchTerm: userEnteredSearchTerm,
+        searchResults: searchResults,
```

Comment:
See comment below about deleting `searchResultsCard.ejs`; this prop name needs to be changed if my suggestion gets implemented.